notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -824,15 +747,15 @@
       },
       "outputs": [],
       "source": [
-        "# Warning: Setting this to true will delete everything in your bucket\n",
+        "# Warning: Setting this to true  delete everything in your bucket\n",
```

Comment:
remove extra space and and change "delete" to "deletes":

- Warning: Setting this to true deletes everything in your bucket